feature-libs/cart/quick-order/core/services/quick-order.service.ts

```diff
@@ -195,15 +196,21 @@ export class QuickOrderService implements QuickOrderFacade {
     let evt: QuickOrderAddEntryEvent = {
```

Comment:
can we change it to `const`? 🤔 
I see that we only mutate the variable, and re-assign it.